torch/csrc/utils/cpp_stacktraces.cpp

```diff
@@ -47,9 +47,31 @@ bool get_cpp_stacktraces_enabled() {
   return enabled;
 }
 
-bool get_disable_addr2line() {
-  static bool disabled = compute_disable_addr2line();
-  return disabled;
+static torch::unwind::Mode compute_symbolize_mode() {
+  auto envar_c = std::getenv("TORCH_SYMBOLIZE_MODE");
```

Comment:
we already ensure this env is set to something? or where is the default provided